pkg/mcs/resourcemanager/server/token_buckets.go

```diff
@@ -31,20 +31,52 @@ const (
 )
 
 const (
-	defaultReserveRatio    = 0.5
-	defaultLoanCoefficient = 2
-	maxAssignTokens        = math.MaxFloat64 / 1024 // assume max client connect is 1024
-	slotExpireTimeout      = 10 * time.Minute
+	defaultBurstLimitFactor = 8.0
```

Comment:
I wonder if this is not suitable for high rate limit settings.

Reply:
How about 2 or 4? Maybe we need a logarithmic-like function rather than a simple multiplication?

Reply:
yes, logarithmic-like is more reasonable.